Sources/Extensions/AppKit/NSViewExtensions.swift

```diff
@@ -101,6 +101,21 @@ public extension NSView {
 			layer?.shadowRadius = newValue
 		}
 	}
+    
+    /// SwifterSwift: set and get background color of NSView
+    @IBInspectable public var backgroundColor: NSColor? {
+        get {
+            if let colorRef = self.layer?.backgroundColor {
```

Comment:
I'm not reviewer, but I have a suggestion. You should use `self` only when required by the compiler. This library also follows this style.

Reply:
hi @VincentSit , removed self, thanks for suggestion!
I never used this standard's before, Can someone help here for understanding how all it works!
I see how codecove and swifter bot for different tasks, I never used it.